### PR TITLE
UK TL: Items

### PR DESCRIPTION
**Pull request by tsunamistate**
@SourceCodeDeleted please review and provide help.
I have marked TLs where I need help as `#, fuzzy`
Best reviewed per commit, starting for 2nd one.

Most of the item stuff except for Uniques

TL decisions:

- Club - Палиця
- Cleaver - Сікач
- Ring - Перстень
- Potion - Зілля
- Great - Величезн(ий/а)
- Large - Велик(ий/а)

Need to file issues:

- Gendering for item prefixes: https://github.com/diasurgical/devilutionX/issues/3563
- Using `drop` debug command results in xstring assert: https://github.com/diasurgical/devilutionX/issues/3562

### CODE REVIEW

File: Translations/uk.po [hunk 1]
```diff
@@ -1835,1351 +1835,1371 @@ msgstr "Не зміг створити головне вікно"
...
+#, fuzzy
 msgid "Plate"
-msgstr ""
+msgstr "Лати"
 
 #: Source/itemdat.cpp:82
+#, fuzzy
 msgid "Splint Mail"
-msgstr ""
+msgstr "Лати з Осколків"
```
Comment (tsunamistate) [inline Translations/uk.po:2125]: Will be changed to "Шинні Лати"
https://uk.wikipedia.org/wiki/%D0%A8%D0%B8%D0%BD%D0%BD%D0%B8%D0%B9_%D0%B7%D0%B0%D1%85%D0%B8%D1%81%D1%82_%D0%BA%D1%96%D0%BD%D1%86%D1%96%D0%B2%D0%BE%D0%BA

**Comment by SourceCodeDeleted** [comment]
I am going through these right now. Some are somewhat interesting

File: Translations/uk.po [hunk 2]
```diff
@@ -1835,623 +1835,636 @@ msgstr "Не зміг створити головне вікно"
...
 #: Source/itemdat.cpp:47 Source/itemdat.cpp:404
 msgid "Griswold's Edge"
-msgstr ""
+msgstr "Лезо Грізволда"
 
 #: Source/itemdat.cpp:48 Source/itemdat.cpp:405
+#, fuzzy
 msgid "Bovine Plate"
-msgstr ""
+msgstr "Свині Лати"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1968]: How do you think about the following? 
бичачий лати

Коров'ячий exists also but it seems weird .

File: Translations/uk.po [hunk 3]
```diff
@@ -1835,623 +1835,636 @@ msgstr "Не зміг створити головне вікно"
...
 #: Source/itemdat.cpp:43 Source/itemdat.cpp:114
 msgid "Scroll of Town Portal"
-msgstr ""
+msgstr "Згорток Порталу в Місто"
 
 #: Source/itemdat.cpp:44 Source/itemdat.cpp:403
+#, fuzzy
 msgid "Arkaine's Valor"
-msgstr ""
+msgstr "Хоробрість Аркейну"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1951]: Looking at the definition of valor , it seems to be more of 'worth'.

He has shown his worth. He has shown his valor.

The translation I found was доблесть ,how do you think ?

File: Translations/uk.po [hunk 4]
```diff
@@ -1835,623 +1835,636 @@ msgstr "Не зміг створити головне вікно"
...
 #: Source/itemdat.cpp:80
 msgid "Scale Mail"
-msgstr ""
+msgstr "Лускова Кольчуга"
 
 #: Source/itemdat.cpp:81
+#, fuzzy
 msgid "Breast Plate"
-msgstr ""
+msgstr "Нагрудні Лати"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:2114]: I am tempted on just calling this Нагрудник. To remove any adjectives. I think in English breastplate is ineword too

File: Translations/uk.po [hunk 5]
```diff
@@ -1835,623 +1835,636 @@ msgstr "Не зміг створити головне вікно"
...
+#, fuzzy
 msgid "Plate Mail"
-msgstr ""
+msgstr "Пластинчасті Лати"
 
 #: Source/itemdat.cpp:84
+#, fuzzy
 msgid "Field Plate"
-msgstr ""
+msgstr "Польові Лати"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:2135]: I looked at difference between field plate and plate mail and they appear to be the same just field plate is lighter.

File: Translations/uk.po (hunk 2 above)
Comment (tsunamistate) [reply Translations/uk.po:1968]: Sure, but you're applying the wrong case to the word
It would be "Бичачі Лати"

File: Translations/uk.po (hunk 3 above)
Comment (tsunamistate) [reply Translations/uk.po:1951]: Sure, that can work

File: Translations/uk.po (hunk 4 above)
Comment (tsunamistate) [reply Translations/uk.po:2114]: OK

File: Translations/uk.po (hunk 5 above)
Comment (tsunamistate) [reply Translations/uk.po:2135]: Let's leave it like that then

**Review by AJenbo — APPROVED** [review]

**Pull request merged**